resources/js/Pages/Vault/Contact/Show.vue

```diff
@@ -295,7 +294,7 @@ const navigateToSelected = () => {
             <!-- family summary -->
             <div v-if="data.group_summary_information.length > 0">
               <div class="mb-6 flex rounded-sm border border-gray-200 p-3 dark:border-gray-700">
-                <img src="/img/group.svg" class="me-2 h-6 w-6" />
+                <img src="/img/group.svg" class="me-2 h-6 w-6" alt="Group" />
```

Comment:
You need to localize the `alt`
```suggestion
                <img src="/img/group.svg" class="me-2 h-6 w-6" :alt="$t('Group')" />
```